roadmap.md

```diff
@@ -1,145 +1,45 @@
 
-We shipped .NET Core 2.0 and .NET Standard 2.0 - you can try it by installing [Visual Studio 2017 (15.3)][vs2017] and [.NET Core 2.0 SDK][Core20Sdk].
+The .NET Core team is currently focussed on Web and console apps. We encourage the community to work with us to improve web and console apps and to extend .NET Core for other scenarios.
 
-[vs2017]: https://www.visualstudio.com/downloads
-[Core20Sdk]: https://www.microsoft.com/net/download/core
+## Released Versions
 
-## Ship Dates
+[.NET Core 2.0](https://github.com/dotnet/core/issues/812) shipped on August 14th, 2017.
+
+For released versions of the product:
+
+* [.NET Core Release Notes](release-notes/README.md)
+* [.NET Core Supported OS Lifecycle Policy](os-lifecycle-policy.md).
+
+## Upcoming Ship Dates
 
 | Milestone                 | Release Date |
 |---------------------------|--------------|
-| .NET Core 2.0 & .NET Standard 2.0 | [Released on 2017/8/14](https://github.com/dotnet/core/issues/812), see [announcement](https://blogs.msdn.microsoft.com/dotnet/2017/08/14/announcing-net-core-2-0/) |
-| UWP 6.0 (implements .NET Standard 2.0) | Q4 2017 ([Win10 Fall Creators Update](https://www.microsoft.com/en-us/windows/upcoming-features)) |
 | .NET Core 2.1 Preview | Q4 2017 |
 | .NET Core 2.1 | Q1 2018 |
 
-Details on sub-milestones are also captured in milestone details of our repos, e.g. [CoreFX repo milestones](https://github.com/dotnet/corefx/milestones).
-
-# Components
-
-.NET Core is a general purpose, modular, cross-platform and open source implementation of .NET.
-It includes a runtime, framework libraries, compilers and tools that support a variety of chip and OS targets.
-These components can be used together or separately.
-
-Major .NET Core components are listed below.
-Please note that this is not meant to be an exhaustive list.
-
-* [Runtime](https://github.com/dotnet/coreclr)
-* [Libraries](https://github.com/dotnet/corefx)
-* [.NET Standard](https://github.com/dotnet/standard)
-* [C#/VB compiler](https://github.com/dotnet/roslyn)
-* [F# compiler](https://github.com/microsoft/visualfsharp)
-* [SDK](https://github.com/dotnet/sdk)
-* [CLI tools](https://github.com/dotnet/cli) / [CLI documentation](https://docs.microsoft.com/en-us/dotnet/articles/core/tools/index)
-* [NuGet](https://github.com/NuGet/Home)
-* [ASP.NET](https://github.com/aspnet)
-* [MSBuild](https://github.com/microsoft/msbuild)
-* [Docker Images](https://github.com/dotnet/dotnet-docker)
-
-This roadmap is intended to communicate project priorities for evolving and extending the scope of .NET Core.
-
-# Supported OS Versions
-
-## .NET Core 2.1 - Supported OS Versions
-
-Proposal: TBD
-
-## .NET Core 2.0 - Supported OS Versions
-
-Proposal:
-
-OS                            |Version                        |Architectures  |Notes
-------------------------------|-------------------------------|---------------|-----
```

Comment:
Where does the whole support matrix live now?

Reply:
Click on the ".NET Core Supported OS Lifecycle Policy" link.